cherab/openadas/repository/atomic.py

```diff
@@ -108,6 +108,47 @@ def update_recombination_rates(rates, repository_path=None):
         _update_and_write_adf11(species, rate_data, path)
 
 
+def add_thermalchargeexchange_rate(species, charge, rate, repository_path=None):
+    """
+    Adds a single thermal charge exchange rate to the repository.
+
+    If adding multiple rates, consider using the update_recombination_rates()
+    function instead. The update function avoids repeatedly opening and closing
+    the rate files.
+
+    :param repository_path:
+    :return:
+    """
+
+    update_thermalchargeexchange_rates({
+        species: {
+            charge: rate
+        }
+    }, repository_path)
+
+
+def update_thermalchargeexchange_rates(rates, repository_path=None):
```

Comment:
Rename to be consistend with the add method.

Reply:
[renamed](https://github.com/Mateasek/cherab_openadas/blob/e9458e956af735110764c7f8d3c7403889e7d9de/cherab/openadas/repository/atomic.py#L130) to:

``` python
def update_thermal_cx_rates(rates, repository_path=None):
```